tutorials/CIFAR_Tutorial.py

```diff
@@ -0,0 +1,187 @@
+import argparse
```

Comment:
Can you add a brief blurb describing this tutorial (a) as a header comment in this file, and (b) in `tutorials/README.md`?